### ISSUE TITLE
[Bug] Round Robin on multi-output pipes not working as intended.

### ISSUE THREAD

**Issue by MCCLXXXVII** [Ellpeck/PrettyPipes#195]
https://github.com/Ellpeck/PrettyPipes/assets/129862749/ea7129c0-35d4-497b-a1cf-77927fb27865

If an extract module is placed before multiple inventories are connected to a single pipe, only the inventories on one side are considered for input. Only once the side is full is the other side considered. See video attached.

As far as I can tell, North/West are favored, haven't tested Up/Down or tri-input pipes.

**Comment by Ellpeck** [comment]
Duplicate of #167
